pages/api/cck/route.ts

```diff
@@ -26,7 +27,12 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     const query = req.query;
     const { date, ref, passcode } = query;
 
-    if(passcode) {
+    const daysDiff = daysSince(parseDashedDate(date as string));
```

Comment:
Same comment as in the other file

Reply:
Updated to throw error and return 400 if date cannot be successfully parsed